pkg/telemetry/statsworker.go

```diff
@@ -34,18 +87,22 @@ func newStatsWorker(
 		roomName:      roomName,
 		participantID: participantID,
 
-		outgoingPerTrack: make(map[livekit.TrackID][]*livekit.AnalyticsStat),
-		incomingPerTrack: make(map[livekit.TrackID][]*livekit.AnalyticsStat),
+		outgoingPerTrack: make(map[livekit.TrackID]Stats),
+		incomingPerTrack: make(map[livekit.TrackID]Stats),
 	}
 	return s
 }
 
 func (s *StatsWorker) appendOutgoing(trackID livekit.TrackID, stat *livekit.AnalyticsStat) {
-	s.outgoingPerTrack[trackID] = append(s.outgoingPerTrack[trackID], stat)
+	stats := s.outgoingPerTrack[trackID]
```

Comment:
Before my previous change, there was another map to maintain what needs to be deleted which was populated from `UnpublishTrack` event and cleaned up in the update. Do we need to bring that back?